### PR TITLE
Cleans up the chem synths jesus christ what the fuck

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Dax Dupont
fix: Chem synths overlays aren't all kinds of fucked anymore.
refactor: Unfucked all of the remaining chem synth code.
refactor: Reagents no longer have an unused var that's always set to true. Who did this?
/:cl:

[why]: Jesus christ it was a mess.
It's unfucked now.

### CODE REVIEW

**Comment by DaxDupont** [comment]
@81Denton why

File: code/modules/reagents/chemistry/reagents.dm [hunk 1]
```diff
@@ -22,7 +22,6 @@
 	var/volume = 0									//pretend this is moles
 	var/color = "#000000" // rgb: 0, 0, 0
 	var/can_synth = TRUE // can this reagent be synthesized? (for example: odysseus syringe gun)
-	var/can_synth_debug = TRUE // can this reagent be synthesized by the debug chem synthesizer?
```
Comment (DaxDupont) [inline code/modules/reagents/chemistry/reagents.dm]: @81Denton why

Comment (81Denton) [reply code/modules/reagents/chemistry/reagents.dm]: don't drink and code kids

Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents.dm]: TBF I too would expect all chems to be synth'd by the debug chem synthesizer.

File: code/modules/reagents/chemistry/machinery/chem_synthesizer.dm [hunk 2]
```diff
@@ -5,44 +5,23 @@
...
+	use_power = NO_POWER_USE
 	var/static/list/shortcuts = list(
 		"meth" = "methamphetamine",
 		"tricord" = "tricordrazine"
 	)
-	var/mutable_appearance/top_overlay
-
-/obj/machinery/chem_dispenser/chem_synthesizer/Initialize()
-	. = ..()
-	GLOB.poi_list += src
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/machinery/chem_synthesizer.dm]: Only a botany/chem main would think to do this so if either got a scrying orb they'd know when to steal it / where to get it back from.

**Comment by ExcessiveUseOfCobblestone** [comment]
Also I bet your PR title violates Githubs ToS

**Comment by DaxDupont** [comment]
Swearing is fine

File: code/modules/reagents/chemistry/machinery/chem_synthesizer.dm (hunk 2 above)
Comment (AlexTheSysop42) [reply code/modules/reagents/chemistry/machinery/chem_synthesizer.dm]: pointless anyway since these are unobtainable

it's just a holdover from SCP-294

**Comment by SpaceManiac** [comment]
Code cleanup looks good.

Would you feel comfortable making the chem ID and amount input fields in the HTML, ORM-style? If not I'll just do it eventually.

**Comment by DaxDupont** [comment]
I've put all the effort I want to spend on this on it. It's admin spawn only so

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**